src/main/java/org/sdnplatform/sync/ISyncService.java

```diff
@@ -133,4 +137,11 @@ public void registerPersistentStore(String storeName, Scope scope)
                        IInconsistencyResolver<Versioned<V>> resolver)
                                throws UnknownStoreException;
 
+
+    
+    
```

Comment:
@tulioalberton could you add some Javadoc style comments to describe what these new ISyncService functions do and return?